main_test.go

```diff
@@ -912,7 +917,7 @@ func testPkgBackend(t *testing.T, pyvm string, table pkg) {
 	if err != nil {
 		t.Fatalf("[%s:%s]: could not create workdir: %v\n", pyvm, table.path, err)
 	}
-	defer os.RemoveAll(workdir)
+	//defer os.RemoveAll(workdir)
```

Comment:
could we retain the defer in place?
(I assume you removed it for debugging)

perhaps modify it to only apply the `os.RemoveAll` if everything went fine?